### PR TITLE
Content update

### PR DESCRIPTION
**Pull request by abdulrauf11**
This PR contains all the content updates that were discussed at the start of this milestone.

## Media file size issue
One possible issue would be using large files on git, which it doesn't handle too well.

We get the following `warning` on some of the files:
```
remote: warning: File static/media/quizintro_video_en.mp4 is 51.09 MB; this is larger than GitHub's recommended maximum file size of 50.00 MB
remote: warning: GH001: Large files detected. You may want to try Git Large File Storage - https://git-lfs.github.com.
```

Possible solutions:
- Host files somewhere else and use URLs in the JSON files.
- Use **git LFS** (needs to be explored further)
- Ignore media files ✅  (we have decided to go with this)

## Files ignored
The following files were ignored in this build:
```
# media files
*.m4v
*.mp4
*.mp3
*.wav
*.ogg
*.jpg
```

## Image optimization
Some of the images are ~1MB in size and can (possibly) be further compressed. It needs to be tested if the images are still taking time to load in production.

## Test link
There is no test link for this PR.

### CODE REVIEW

File: src/content/en/quiz.json [hunk 1]
```diff
@@ -9,22 +9,22 @@
...
+              "/media/s1_q1_image_1.jpg",
+              "/media/s1_q1_image_2.jpg",
+              "/media/s1_q1_image_3.jpg"
             ]
           }
         },
         "solution": {
           "correctOption": 1,
-          "text": "The earliest bionic eyes provided very limited vision. Users could distinguish light and shadow, but not much else.",
+          "text": "The earliest bionic eyes provided very limited vision. Users could distinguish light and shadow, but not much else.Early bionic eyes provided very limited vision. Users could distinguish light and shadow, but not much else.",
```
Comment (tnordberg) [inline src/content/en/quiz.json]: Could you add a space between `else.` and `Early`?

File: static/audio/.gitkeep [hunk 2]
```diff
@@ -0,0 +1 @@
+# Tell git to track this directory, even when empty
```
Comment (tnordberg) [inline static/audio/.gitkeep:1]: Thanks for adding these `.gitkeep`s.

File: src/content/en/quiz.json [hunk 3]
```diff
@@ -409,16 +521,83 @@
...
+
+      {
+        "questionIntro": {
+          "title": "Question eight",
+          "audio": "/audio/s3_i8_audio_en.mp3",
+          "captions": "/captions/s3_i8_audio_en.vtt"
+        },
+        "question": {
+          "type": "multi-choice-text",
+          "text": "Bionic eyes use electrodes to stimulate nerves in the eye or brain.  How many electrodes would you need to reproduce “natural” vision? ",
```
Comment (tnordberg) [inline src/content/en/quiz.json]: Use single space between `brain.` and `How`.

**Review by tnordberg — CHANGES_REQUESTED** [review]
Hey @abdulrauf11 – I found a couple minor typos and left line comments. Could you address them (even if the typos came from our doc). 

Otherwise, looks great!

File: static/audio/.gitkeep (hunk 2 above)
Comment (abdulrauf11) [reply static/audio/.gitkeep:1]: You're welcome!

**Comment by abdulrauf11** [comment]
> Hey @abdulrauf11 – I found a couple minor typos and left line comments. Could you address them (even if the typos came from our doc).
> 
> Otherwise, looks great!

@tnordberg  Done!

**Review by tnordberg — APPROVED** [review]

**Pull request merged**